pages/common/source.md

```diff
@@ -0,0 +1,7 @@
+# source
+
+> Evaluate contents of file.
```

Comment:
How about `A shell builtin command to executes the content of the file passed as argument`

Reply:
There is another explanation from bash shell: `Execute commands from a file in the current shell.` What do you think about that?

Reply:
I like the latter there :-)